openfl/component/aggregator/aggregator.py

```diff
@@ -990,6 +1051,7 @@ def _end_of_round_check(self):
 
         self.round_number += 1
         # resetting stragglers for task for a new round
+        #E.L should it be saved?
```

Comment:
To be on the safe side - yes. This variable reflects the aggregator's knowledge of late/straggler collaborators. Otherwise, I'm not sure how the aggregator would behave if there are stragglers while it's recovering.

Reply:
+1 . Straggler list does not need to be persisted.

Reply:
Thanks for the clarification